gamemodes/terrortown/gamemode/client/cl_keys.lua

```diff
@@ -76,29 +77,38 @@ function GM:PlayerBindPress(ply, bindName, pressed)
             return TBHUD:UseFocused()
         end
 
-        local tr = util.TraceLine({
-            start = ply:GetShootPos(),
-            endpos = ply:GetShootPos() + ply:GetAimVector() * 100,
-            filter = ply,
-            mask = MASK_SHOT,
-        })
-
-        if not tr.Hit or not IsValid(tr.Entity) then
-            return
-        end
-
+        -- Find out if a marker is focussed otherwise check normal use
         local isClientOnly = false
-        if isfunction(tr.Entity.ClientUse) then
-            isClientOnly = tr.Entity:ClientUse()
+        local useEnt = markerVision.GetFocussedEntity()
+        local isRemote = IsValid(useEnt)
```

Comment:
maybe something like `isRemoteMarker` so that you see that this has something to do with marker vision?

Reply:
Not sure if that is relevant. This internally just remotes use an entity, that is remote controlled. So the moment we get the focussed entity from markervision we dont do anything with the marker anymore, so we only need to know if this is a remote entity or a close one.

Reply:
I just thought it would make it easier at a later point to read the code

Reply:
Not sure I understand that. Because the variable isRemote does not contain the marker entity, but a boolean to detect if we should trigger RemoteUse or normal ClientUse + serverside Use.

Reply:
I guess you're right